xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -639,6 +639,20 @@
       }
     }
   },
+  "Files.SetFileDetails": {
+    "type": "method",
+    "description": "Update the given specific file with the given details",
+    "transport": "Response",
+    "permission": "UpdateData",
+    "params": [
+      { "name": "file", "type": "string", "required": true, "description": "Full path to the file" },
+      { "name": "media", "$ref": "Files.Media", "default": "video", "required": true, "description": "File type to update correct database. Only video is supported." },
```

Comment:
Maybe change it to `Currently only \"video\" is supported.` to indicate that this might change in the future.

And `default` doesn't make any sense for a required parameter/property.